lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -96,6 +96,10 @@
               offer, sku and version. NOTE: set image.version to 'latest' to get the most recent version of a given
               image."
         required: true
+    availability_set:
+        description:
+            - Name or ID of an existing availability set to add the VM to. None by default.
```

Comment:
add:
    default: null

Reply:
Also we should tell user that this availability set should be in the same resource group of current vm.

Reply:
fixed